src/engine/renderer/tr_shader.cpp

```diff
@@ -74,6 +77,13 @@ static const vec3_t dxNormalFormat = {  1.0, -1.0,  1.0 };
 static Cvar::Cvar<bool> r_dpMaterial("r_dpMaterial", "Enable DarkPlaces material compatibility", Cvar::LATCH, false);
 Cvar::Cvar<bool> r_dpBlend("r_dpBlend", "Enable DarkPlaces blend compatibility, process GT0 as GE128", Cvar::NONE, false);
 
+static std::string padStringSuffix( std::string const &str, int num, char pad = ' ' )
```

Comment:
You can do left-aligned columns with printf-style formatting. `Log::Notice("%-9s", str)` for a width-9 column. Or `Log::Notice("%-*s", 9, str)` is the same but with width determined at runtime.

Reply:
Done. I haven't found a way to use a custom padding character but for writing the line separator I used a constructor instead, doing `std::string lineSeparator( header.length(), '-' );`.